src/sentry/eventtypes/transaction.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+
+from sentry.utils.safe import get_path
+
+from .base import BaseEvent
+
+
+class TransactionEvent(BaseEvent):
+    key = 'transaction'
+
+    def get_metadata(self, data):
+        description = get_path(data, 'contexts', 'trace', 'description')
+        transaction = get_path(data, 'transaction')
+        return {
+            'title': description or transaction,
+            'location': transaction
+        }
+
+    def get_title(self, metadata):
```

Comment:
I thought a bit about this stuff and wonder if we even want to do anything with title and location for transaction events. Seems like we could cut past that

Reply:
If this currently means that title can be empty we need to find some fallback. If the title is empty the event is not clickable in the events view.

Reply:
Title falls back to transaction because of this issue. We should always have a transaction.

Events view is a good point. I was thinking purely about issue details page where we won't be present anyway